images/notary/Dockerfile

```diff
@@ -3,6 +3,7 @@ MAINTAINER Chris Kotfila <chris.kotfila@kitware.com>
 
 RUN apk add --no-cache \
         bash \
+        git \
```

Comment:
Was this discovered by running a simulated protected pipeline on the packages repo?

Reply:
yup, ref. https://gitlab.spack.io/spack/spack-packages/-/jobs/16971601